docs/src/main/tut/datatypes/either.md

```diff
@@ -212,132 +212,6 @@ magic("123") match {
 }
 ```
 
-## Either in the small, Either in the large
```

Comment:
@adelbertc I've come up with a new version of docs. Only some parts that had been no longer true in presence of variance were rewritten. Some other parts were rearranged. Please check if you like it. ( Since Contributors' Guide discourages squashing commits, it may be awkward to compare :-( )